pkg/operator/ceph/object/controller_test.go

```diff
@@ -330,4 +330,28 @@ func TestCephObjectStoreController(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, cephv1.ConditionReady, objectStore.Status.Phase, objectStore)
 	logger.Info("PHASE 3 DONE")
+
+	// Test the functionality of verifyObjectUserCleanup
+	// function returns true if there are no cephobjectusers,
+	// otherwise false
```

Comment:
We need 2 tests, one, when there is/, are CephObjectStoreUsers and one when there is none.

Reply:
Done